.env.appStore.example

```diff
@@ -53,12 +53,12 @@ SLACK_CLIENT_ID=
 SLACK_CLIENT_SECRET=
 
 #   - STRIPE
-NEXT_PUBLIC_STRIPE_PUBLIC_KEY= # pk_test_...
-STRIPE_PRIVATE_KEY=            # sk_test_...
-STRIPE_WEBHOOK_SECRET=         # whsec_...
-STRIPE_CLIENT_ID=              # ca_...
-PAYMENT_FEE_FIXED=10           # Take 10 additional cents commission
-PAYMENT_FEE_PERCENTAGE=0.005   # Take 0.5% commission
+NEXT_PUBLIC_STRIPE_PUBLIC_KEY=        # pk_test_...
+STRIPE_PRIVATE_KEY=                   # sk_test_...
+STRIPE_WEBHOOK_SECRET=                # whsec_...
+STRIPE_CLIENT_ID=                     # ca_...
+STRIPE_PAYMENT_FEE_FIXED=10           # Take 10 additional cents commission
+STRIPE_PAYMENT_FEE_PERCENTAGE=0.005   # Take 0.5% commission
```

Comment:
Can we revert these name changes? I know this should be named `STRIPE_` but we are going to prioritize backwards compatibility in this case. Not doing so will cause much unexpected behavior for self-hosted users. Doing so will allow us to merge without user action needed.